### DISCUSSION TITLE
Hiring Serenity developer

### DISCUSSION THREAD

**Discussion by RamziTanani** [serenity-is/Serenity#5853] [Help] [answered]
Hi,

Hopefully, this is not the wrong place to post this. We have plans to migrate a Codeigniter-based Education ERP to .Net Core (well, basically means developing it again). We've been delaying this as it is a scary, time-consuming, and expensive process.

We're joining the Serenity party late it seems. Looks interesting. We want to try this first with a small system for job training. This will give us an idea of how much serenity to expect if we go for it with the big one (The ERP).

Gosh, here I am rambling again...

Basically, looking for a developer who is well-drilled in Serenity to develop a small training system :)

**Comment by VictorTomaili** [comment, marked as answer]
Contact us via [sales@serenity.is](mailto:sales@serenity.is) 
https://serenity.is/

**Reply by RamziTanani** [reply]
Thanks. Sent already.

**Comment by luongnv6** [comment]
Hi, @RamziTanani 
I have been using Serenity for more than 3 years, please contact me
luongit.2007@hotmail.com.vn

**Comment by hairangsua** [comment]
Hi, @RamziTanani 
I have been using Serenity for more than 3 years, please contact me
hoangphihai@gmail.com
